README.md

```diff
@@ -190,6 +190,35 @@ $dumpCommand = MySql::create()
     ->dumpToFile('dump.sql.gz');
 ```
 
+There are three compressors ```gzip```, ```bzip2``` and ```lzma``` all uses maximum compression levels.
+
+```php
+$dumpCommand = MySql::create()
+    ->setDbName('dbname')
+    ->setUserName('username')
+    ->setPassword('password')
+    ->GzipCompression()
```

Comment:
I don't want dedicated methods for choosing the compression. Instead we should pass objects.

Refactor `GzipCompression()` to something like `useCompression(new GzipCompression)`

Reply:
Done!